include/unk_020040F4.h

```diff
@@ -65,8 +66,8 @@ u8 FUN_020048D0(u16 param0);
 u32 GF_SndPlayerGetSeqNo(u32 *param0);
 u32 FUN_020048F4(u16 param0);
 u16 FUN_02004900(u32 param0);
-u32 GF_MIC_StartAutoSampling(u32 param0);
-u32 GF_MIC_StopAutoSampling(u32 param0);
+u32 GF_MIC_StartAutoSampling(struct MIC_SamplingData *param0);
+u32 GF_MIC_StopAutoSampling();
```

Comment:
void in parameter list (u32? this is a bit confusing for me)

Reply:
I removed the u32 to replace it with a struct. Or are you talking about the return type?

Reply:
no I mean, in GF_MIC_StopAutoSampling you replaced the u32 with a blank parameter list, so I'm unsure what should be there, void or a u32, or something else

Reply:
This was a mistake I did when decompiling `GF_MIC_StopAutoSampling`. the argument wasn't even used and now that I decompiled code calling it, it's clear it just doesn't have any argument.